sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -32,6 +32,7 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+#define TCB_EQUAL(a,b)      ((a == b) ? 1 : 0)
```

Comment:
This can just be

```c
#define TCB_EQUAL(a, b)     ((a) == (b))
```

(also mind the parenthesis I put in there, otherwise weird evaluation errors might pop up)